src/__tests__/pageObject/pageObject.test.js

```diff
@@ -48,3 +67,15 @@ describe("create locators names", () => {
     expect(createLocatorNames(elementsWithoutNames, ElementLibrary.MUI)).toStrictEqual(elementsWithNames);
   });
 });
+
+describe("pageObjectTemplate", () => {
+  describe("should return pageObjectHTML with FindBy import", () => {
+    templateTestDataWithFindBy.forEach(({ input, output }) => {
```

Comment:
it can be moved inside `describe("page object code generation"`, because it covers same function